src/kaltura-player.js

```diff
@@ -872,7 +874,7 @@ class KalturaPlayer extends FakeEventTarget {
     this.dispatchEvent(new FakeEvent(VISIBILITY_CHANGE, {visible: this._isVisible}));
 
     if (this.config.playback.autoplay === AutoPlayType.IN_VIEW && this._isVisible && !this._playbackStart) {
-      this._localPlayer.autoPlay();
+      this._localPlayer.play(true);
```

Comment:
this should be using the new config object

Reply:
Right!!!

Reply:
Missed that as it worked also with the buggy code because it doesn't require me gesture on my chrome.
Fixed